### PR TITLE
EC minutes for 28 May 2024

### PR DESCRIPTION
**Pull request by jasongrout**
EC minutes for 28 May 2024

### CODE REVIEW

**Comment by Ruv7** [comment]
As a best practice I would propose that the EC establish time in our weekly meetings to approve the minutes from the previous meeting. This is a standard practice in many orgs - committee approval public minutes. The bar of having one other person approve before committing is not high enough.

**Comment by jasongrout** [comment]
I agree we should make approval more official.

For last week, I was assuming that us working on the minutes together in the google doc was enough of a "EC approval", and then the review here was to check my transcription, so not as high of a bar. This week I think I reworked some of the text in the document after the meeting, so I requested more reviews. But even then, I agree we should make the approval more explicit.

Do you think approval via the PR review mechanism here is sufficient for this week? Or should we take up the topic next week in our actual meeting?

**Comment by Ruv7** [comment]
I agree with last week being ok as is because there were not changes or additions post the meeting that I'm aware of. I do think baking it into the process of creating the minutes does two things: 1. ensures it happens in a consistent/timely manner 2. Allows for quick clarification/discussion/changes. I think it's fine to review asynch and that could speed up the vote. However I would say that we don't merge the minutes until the weekly meeting. If a majority reviews before the meeting in an  asynchronous manner that will speed things up.

**Comment by jasongrout** [comment]
So what I'm hearing is the following process:

1. (Ideally) Create the public meeting minutes together in the last few minutes of our meeting, summarizing our discussion and internal meeting minutes.
2. Open a PR with the meeting minutes (which may have some post-meeting editing), ask the rest of the EC for a PR review.
3. (Ideally) EC members approve the GitHub PR before the next EC meeting.
4. Officially approve the minutes at the next EC meeting (ideally this is quick if everyone has already approved them on GitHub)
5. Merge the meeting minutes PR.

Is that what you had in mind?

**Comment by Ruv7** [comment]
I agree except where 1 says ideally. I would simply say that is the required process. Any exceptions should be infrequent and explained.

**Review by Zsailer — APPROVED** [review]

**Comment by Zsailer** [comment]
Thank you @jasongrout and @Ruv7 for fleshing out a concrete process for minute keeping. 🚀 I love where y'all landed.

File: docs/meeting_notes/2024.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+- Fernando Perez
+- Zach Sailer
+- Afshin Darian
+- Ana Ruvalcaba
+
+### Notes
+
+* Continued work on creating a PR for the EC/SSC voting about moving to Linux Foundation
+* Met with Linux Foundation folks to ask questions regarding the Jupyter Foundation and its governing body, as well as to clarify the entities, relationships, and responsibilities of LF, LF Charities, Project Jupyter, and the Jupyter Foundation.
+* We decided (but have not yet voted on) that the EC can and should raise the quorum requirements for EC votes about governance change even if the decision-making guide provides a low quorum and decision-making threshold.
```
Comment (jasongrout) [inline docs/meeting_notes/2024.md:17]:
```suggestion
* We discussed (but have not yet voted on) the EC raising the quorum and approval requirements for EC votes about governance change even if the decision-making guide provides a low quorum and decision-making threshold.
```

**Review by Zsailer — APPROVED** [review]

**Comment by jasongrout** [comment]
Minutes approved in the 4 June 2024 meeting

**Pull request merged**